src/Components/ConnectLog/LogsTable.js

```diff
@@ -168,7 +170,9 @@ const LogsTable = () => {
   const onClick = (id) => {
     (async () => {
       const data = await configApi.getPlaybook(id);
+      addInfoNotification('Log is being downloaded');
       downloadFile(data);
+      addSuccessNotification('Download successful');
```

Comment:
We should add an `addErrorNotification` here. If I replace `const data = await configApi.getPlaybook(id);` with `const data = await configApi.getPlaybook('blah');`, it will not fetch. The error returns an object with each character of a string separated into individual keys. It's annoying and I'm assuming this is a backend issue.

For now, I think we can simply replace the `onClick` function with this:
```
  const onClick = async (id) => {
    try {
      const data = await configApi.getPlaybook(id);
      addInfoNotification('Log is being downloaded');
      downloadFile(data);
      addSuccessNotification('Download successful');
    } catch (error) {
      let errorString = '';
      for (const value of Object.values(error)) {
        errorString += value;
      }
      addErrorNotification('Download failed', { description: errorString });
    }
  };
```

Reply:
thanks for the tip, missed that